src/floating-focus.js

```diff
@@ -180,15 +188,41 @@ export default class FloatingFocus {
 		});
 	}
 
-	repositionElement(target, floater) {
+	standardizeFloat(number) {
+		return Math.round(parseFloat(number.toFixed(3) * 1000)) / 1000;
```

Comment:
3 decimalen? Is 2 niet ook oké?

Reply:
Dit is omdat CSS altijd getallen van > 3 decimalen afrond op 3 decimalen, dus het leek me het keurigst om dat dus aan te houden.

Reply:
Ik weet het niet zo goed.

In het overzicht maakt Chrome er altijd 3 van, maar in de waarden zijn ze weer een andere lengte...

<img width="186" alt="Screenshot 2019-12-16 at 10 46 47" src="https://user-images.githubusercontent.com/147413/70896664-6dec3f00-1ff1-11ea-9a48-d01a9d583bfe.png">
<img width="187" alt="Screenshot 2019-12-16 at 10 46 56" src="https://user-images.githubusercontent.com/147413/70896663-6d53a880-1ff1-11ea-90ec-fbbf748bcdb6.png">
<img width="256" alt="Screenshot 2019-12-16 at 10 46 40" src="https://user-images.githubusercontent.com/147413/70896770-a3912800-1ff1-11ea-888a-8494a1ad1501.png">
<img width="258" alt="Screenshot 2019-12-16 at 10 38 32" src="https://user-images.githubusercontent.com/147413/70896666-6dec3f00-1ff1-11ea-8d7f-9c638b2fb0e7.png">
<img width="254" alt="Screenshot 2019-12-16 at 10 45 47" src="https://user-images.githubusercontent.com/147413/70896665-6dec3f00-1ff1-11ea-8745-71e651cb2f08.png">

Maar laat eerst maar zo.

Reply:
Oh wacht het is gewoon op 6 cijfers significantie... Ja dan is dit niet per se perfect; nouja zolang het werk werkt het I guess